task/buildpacks/0.4/README.md

```diff
@@ -0,0 +1,107 @@
+ - **`PLATFORM_DIR`**: The name of the platform directory. _(optional, default: "empty-dir")_
+
+## Results
+
+ - **`APP_IMAGE_DIGEST`**: The digest of the built `APP_IMAGE`.
+
+## Builders
+
+_The following are the suggested [builders][builders] from the [Cloud Native Buildpacks][buildpacks-io] project. This is only a subset of builders available._
+
+ - **`gcr.io/buildpacks/builder:v1`**: Ubuntu 18 base image with buildpacks for .NET, Go, Java, Node.js, and Python
+ - **`heroku/buildpacks:18`**: Base builder for Heroku-18 stack, based on ubuntu:18.04 base image
+ - **`heroku/buildpacks:20`**: Base builder for Heroku-20 stack, based on ubuntu:20.04 base image
+ - **`paketobuildpacks/builder:base`**: Ubuntu bionic base image with buildpacks for Java, .NET Core, NodeJS, Go, Ruby, NGINX and Procfile
+ - **`paketobuildpacks/builder:full`**: Ubuntu bionic base image with buildpacks for Java, .NET Core, NodeJS, Go, PHP, Ruby, Apache HTTPD, NGINX and Procfile
+ - **`paketobuildpacks/builder:tiny`**: Tiny base image (bionic build image, distroless-like run image) with buildpacks for Java Native Image and Go
+
+[builders]: (https://buildpacks.io/docs/concepts/components/builder/)
+[buildpacks-io]: (https://buildpacks.io)
+
+## Platforms
+
+The Task can be run on `linux/amd64` platform.
+
+## Usage
+
+See the following samples for usage:
+
+ - **[`env-vars.yaml`](samples/env-vars.yaml)**: A PipelineRun configured to provide _build-time_ environment variables.
+ - **[`results.yaml`](samples/results.yaml)**: A PipelineRun configured to demonstrate usage of task results.
+ - **[`dockerconfig.yaml`](samples/dockerconfig.yaml)**: A PipelineRun configured to demonstrate usage of `dockerconfig` workspace for referencing `.docker/config.json` file.
+
+## Support
+
+The [Buildpacks Community](http://buildpacks.io/community/) is always here to help. 
+
+We can be found in our [discussion board][discussion-board] or [slack][slack] (`#tekton`).
+
+[discussion-board]: https://github.com/buildpacks/community/discussions
+[slack]: https://slack.buildpacks.io
+
+## Contributing
+
+We ❤ contributions.
+
+This task is maintained at [buildpacks/tekton-integration](https://github.com/buildpacks/tekton-integration). Issues, pull requests and other contributions can be made there. 
+
+To learn more, read the [CONTRIBUTING][contributing] and [DEVELOPMENT][development] documents.
+
+[contributing]: https://github.com/buildpacks/.github/blob/main/CONTRIBUTING.md
+[development]: https://github.com/buildpacks/tekton-integration/blob/main/DEVELOPMENT.md
+
+## Previous Versions
+
+For support of previous [Platform API][platform-api]s use a previous version of this task.
+
+> Be sure to also supply a compatible builder image (`BUILDER_IMAGE` input) when running the task (i.e. one that has a lifecycle that supports the platform API).
+
+| Version        | Platform API
+|----            |-----
+| [0.3](../0.3/) | [0.3][platform-api-0.3]
```

Comment:
there is no version change from 0.3 to 0.4 so can you please revert this back to 0.2?
```suggestion
| [0.2](../0.2/) | [0.3][platform-api-0.3]
```

Reply:
@vinamra28 isn't this the buildpacks version? if so, this PR bumps from 0.3 to 0.4

Reply:
@zroubalik yeah this is buildpacks version but if you see buildpacks version 0.3 is using platform api 0.4 which will be same in 0.4 version of buildpacks, so no platform API bump whereas buildpacks version 0.2 is using platform api 0.3

Reply:
ack, gotcha :)

Reply:
fixed